app/main/Processors/Excludes.php

```diff
@@ -32,7 +32,7 @@ protected function process(string $target_path, string $filename, string $locale
 
         $content = $this->merge($this->keys, $target, $filename);
 
-        $this->sort($content);
+        $this->sort($content, false);
```

Comment:
No needed because $key is not used in sort method

Reply:
Now [used](https://github.com/andrey-helldar/Laravel-lang/blob/patch/2021-05-26/22-46/app/main/Processors/Processor.php#L130) :)

Reply:
Yes and no, because in Excludes class, you have a sort method which doesn't use the second argument